docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -115,15 +115,15 @@ starter.
 
 [source,java]
 ----
-public Topic createTopic(String topicName)
+public Topic createTopic(String defaultTopicName)
```

Comment:
I think this is coincidental after renaming property in the `GcpPubSubProperties`.
Should be reverted. We really don't talk here about a `default` one.

Thanks

Reply:
agreed, I noticed it after pushing, fix is already in.